### ISSUE TITLE
reflection warnings in websocket code

### ISSUE THREAD

**Issue by tcrayford** [sunng87/ring-jetty9-adapter#21]
Hi

There appear to be a whole bunch of reflection warnings in the websocket code:

```
(set! *warn-on-reflection* true) (require '[ring.adapter.jetty9 :reload true])
true
Reflection warning, ring/adapter/jetty9.clj:51:23 - reference to field getRemote can't be resolved.
Reflection warning, ring/adapter/jetty9.clj:51:23 - call to method sendString can't be resolved (target class is unknown).
Reflection warning, ring/adapter/jetty9.clj:47:5 - reference to field getRemote can't be resolved.
Reflection warning, ring/adapter/jetty9.clj:43:23 - reference to field getRemote can't be resolved.
Reflection warning, ring/adapter/jetty9.clj:43:23 - call to method sendString can't be resolved (target class is unknown).
Reflection warning, ring/adapter/jetty9.clj:39:23 - reference to field getRemote can't be resolved.
Reflection warning, ring/adapter/jetty9.clj:39:23 - call to method sendBytes can't be resolved (target class is unknown).
Reflection warning, ring/adapter/jetty9.clj:89:7 - call to method onWebSocketClose can't be resolved (target class is unknown).
Reflection warning, ring/adapter/jetty9.clj:82:7 - call to method onWebSocketConnect can't be resolved (target class is unknown).
Reflection warning, ring/adapter/jetty9.clj:212:30 - call to org.eclipse.jetty.server.ServerConnector ctor can't be resolved.
Reflection warning, ring/adapter/jetty9.clj:220:33 - call to org.eclipse.jetty.server.ServerConnector ctor can't be resolved.
```

Any chance they can get cleared up? Seems like they could be a potential perf issue for some folk.

**Comment by sunng87** [comment]
Thanks for reporting. I will find some time to fix these warnings.

**Comment by sunng87** [comment]
Closed with 9fbf28b

**Comment by tcrayford** [comment]
Much <3. Thanks so much for this, and for this repo. My app (yellerapp.com) has done several billion http requests through this codebase, and it's never let me down once. Thanks again.
